intermine/intermine
# Added export frictionless data package feature - WIP

## Details

Exports a frictionless [tabular data package](https://specs.frictionlessdata.io/tabular-data-package/#example) with a [tabular data resource](https://specs.frictionlessdata.io/tabular-data-resource/#examples)

This is a work in progress and the data sources information is hardcoded right now.

## Testing

1. Choose any template or build a query.
2. While exporting check the **Compress results** checkbox.
3. Select the **Use Zip compression (produces a .zip archive)** radio button.
4. Click on the **Download file** button.

## Checklist

Before your pull request can be approved, be sure to check all boxes:

- [ ] Passing unit test for new or updated code (if applicable)
- [x] Passes all tests – according to Travis
- [x] Documentation (method documentation, etc.)
- [x] Single purpose
- [x] Well commented code
- [x] Checkstyle
